LICENSE.md

```diff
@@ -9,7 +9,7 @@ Parameters
 
 Licensor: Arcadia Finance
 
-Licensed Work: Arcadia Finance Lending Core. The Licensed Work is (c) 2022 Arcadia Finance
```

Comment:
Change to:
Arcadia Finance - Arcadia Vaults

So it's consistent with the licence of Arcadia lending